bin/autodeploy.sh

```diff
@@ -0,0 +1,416 @@
+ufw allow 9160/tcp;
+ufw allow 7000/tcp;
+ufw allow 7199/tcp;
+
+#elasticsearch
+ufw allow 9300/tcp;
+ufw allow 9200/tcp;
+
+#minio
+ufw allow 9000/tcp;
+ufw allow 9092/tcp;
+
+#rabbitmq
+ufw allow 5671/tcp;
+ufw allow 5672/tcp;
+ufw allow 4369/tcp;
+ufw allow 25672/tcp;
+'"; done
+
+  d helm install cassandra-external ./charts/cassandra-external --values ./values/cassandra-external/values.yaml
+  d helm install elasticsearch-external ./charts/elasticsearch-external --values ./values/elasticsearch-external/values.yaml
+  d helm install minio-external ./charts/minio-external --values ./values/minio-external/values.yaml
+
+  cp values/databases-ephemeral/prod-values.example.yaml values/databases-ephemeral/values.yaml
+  d helm install databases-ephemeral ./charts/databases-ephemeral/ --values ./values/databases-ephemeral/values.yaml
+
+  d helm install fake-aws ./charts/fake-aws --values ./values/fake-aws/prod-values.example.yaml
+  d helm install demo-smtp ./charts/demo-smtp --values ./values/demo-smtp/prod-values.example.yaml
+  d helm install reaper ./charts/reaper
+
+  cp values/wire-server/prod-values.example.yaml values/wire-server/values.yaml
+  sed -i "s/example.com/$TARGET_SYSTEM/g" values/wire-server/values.yaml
+  sed -i "s/# - \"turn:<IP of restund1>:80\"/- \"turn:$HOST_IP:3478\"/g" values/wire-server/values.yaml
+  sed -i "s/# - \"turn:<IP of restund1>:80?transport=tcp\"/- \"turn:$HOST_IP:3478?transport=tcp\"/g" values/wire-server/values.yaml
+
+  d helm install wire-server ./charts/wire-server --timeout=15m0s --values ./values/wire-server/values.yaml --values ./values/wire-server/secrets.yaml
+
+  sed -i "s/example.com/$TARGET_SYSTEM/" values/webapp/prod-values.example.yaml
+  d helm install webapp ./charts/webapp --values ./values/webapp/prod-values.example.yaml
+
+  sed -i "s/example.com/$TARGET_SYSTEM/" values/team-settings/prod-values.example.yaml
+  d helm install team-settings ./charts/team-settings --values ./values/team-settings/prod-values.example.yaml --values ./values/team-settings/prod-secrets.example.yaml
+
+  sed -i "s/example.com/$TARGET_SYSTEM/" values/account-pages/prod-values.example.yaml
+  d helm install account-pages ./charts/account-pages --values ./values/account-pages/prod-values.example.yaml
+
+  cp values/ingress-nginx-controller/prod-values.example.yaml ./values/ingress-nginx-controller/values.yaml
+  d helm install ingress-nginx-controller ./charts/ingress-nginx-controller --values ./values/ingress-nginx-controller/values.yaml
+
+  KUBENODEIP=$(d kubectl get pods -l app.kubernetes.io/name=ingress-nginx -o=custom-columns=IP:.status.hostIP --no-headers)
+  sudo sed -i "s/define KUBENODEIP.*/define KUBENODEIP = $KUBENODEIP/" /etc/nftables.conf
+  sudo systemctl restart nftables
+
+  INGRESSNODE=$(d kubectl get pods -l app.kubernetes.io/name=ingress-nginx -o=custom-columns=NODE:.spec.nodeName --no-headers)
+  d kubectl cordon "$INGRESSNODE"
+
+  wget https://charts.jetstack.io/charts/cert-manager-v1.13.2.tgz
```

Comment:
is there a reason we manually download and unpack charts instead of using `helm repo add`?

Reply:
Taken from https://github.com/wireapp/wire-server-deploy/blob/master/offline/docs_ubuntu_22.04.md

I guess the original thought was to avoid creating external dependencies for a "true" offline install, without internet. Which does not work anyway with (automated) demo WIAB, but I wanted to avoid upstreaming changes to the "official" offline documentation.

Reply:
I think we can remove this manual pulling of cert-manager helm chart in another pr and use our new helm-charts / wire-builds repo to put in the offline artifact.